util/jwtTokenGenerator.js

```diff
@@ -0,0 +1,27 @@
+const CryptoJS = require("crypto-js");
+
+function base64url(source) {
+  // Encode in classical base64
+  let encodedSource = CryptoJS.enc.Base64.stringify(source);
+
+  // Remove padding equal characters
+  encodedSource = encodedSource.replace(/=+$/, "");
+
+  // Replace characters according to base64url specifications
+  encodedSource = encodedSource.replace(/\+/g, "-");
+  encodedSource = encodedSource.replace(/\//g, "_");
+
+  return encodedSource;
+}
+
+function generateJWTToken(header, data, secret) {
+  const stringifiedHeader = CryptoJS.enc.Utf8.parse(JSON.stringify(header));
+  const encodedHeader = base64url(stringifiedHeader);
+  const stringifiedData = CryptoJS.enc.Utf8.parse(JSON.stringify(data));
+  const encodedData = base64url(stringifiedData);
+  const token = `${encodedHeader}.${encodedData}`;
+  const signature = base64url(CryptoJS.HmacSHA256(token, secret));
+  //   signature = base64url(signature);
```

Comment:
Lets remove this commented line if its not used

Reply:
Done